data/crates.json

```diff
@@ -748,20 +748,36 @@
                     ]
                 },
                 {
-                    "slug": "grpc",
-                    "name": "gRPC",
+                    "slug": "rpc",
+                    "name": "RPC",
                     "purposes": [
                         {
                             "name": "General Purpose",
                             "recommendations": [{
                                 "name": "tonic",
                                 "notes": "gRPC over HTTP/2 with full support for asynchronous code. Works with tokio"
                             }]
+                        },
+                        {
+                            "name": "Protocol Buffer",
+                            "recommendations": [{
+                                "name": "prost",
+                                "notes": "Runtime protocol buffer serialization and deserialization"
+                            },
+                            {
+                                "name": "prost-build",
```

Comment:
I don't think mentioning prost-build separately from prost is useful, since most people will use them together. In my mind the goal of this project is leave breadcrumbs that will make it easy for inexperienced users to find popular/good projects, adding more stuff that doesn't serve that goal will then add a kind of bloat that just makes it harder to find stuff?

Reply:
Yeah, that's a good point. I don't mind cutting it, prost very clearly points to prost-build.